src/config/firmware.md

```diff
@@ -21,7 +21,24 @@ Install the AMD package, `linux-firmware-amd`, which contains microcode for both
 AMD CPUs and GPUs. AMD CPUs and GPUs will automatically load the microcode, no
 further configuration required.
 
-## Verification
+### Verification
 
 The `/proc/cpuinfo` file has some information under `microcode` that can be used
 to verify the microcode update.
+
+## Adding firmware
+
+A list of firmware available in the Void repositories can be found by running:
+
+```
+$ xbps-query -Rs firmware | awk '{ print $2 }' | grep firmware
+```
+
+## Removing firmware
+
+By default, `linuxX.Y` packages and the `base-system` package install a number
+of firmware packages. To remove unneeded firmware packages, use `ignorepkg`
+entries in an XBPS configuration file, as described in
+[xbps.d(5)](https://man.voidlinux.org/xbps.d.5), to specify which packages are
+unneeded, then use [xbps-remove(1)](https://man.voidlinux.org/xbps-remove.1) to
+remove them.
```

Comment:
I feel like this needs to make it clearer that it is not necessary to remove the firmware that aren't in use.

And the explanation regarding `xbps.d` seems like something we could mention in the XBPS pages instead? Then make a link here.

Reply:
i'll certainly mention the first point.
Re. the second point, probably, but whereabouts specifically?

Reply:
Not sure how big of a change this would be, but perhaps rename `files-and-packages` from "Finding..." to just "Files and Packages" and adding some tricks there?

Reply:
Hmm, how about "General Package Management" instead? Then it could be the place to address both #201 and #358.

Reply:
The one reason I don't like this as an option was because index already listed the `xbps-*` utilites, which are the main part of package management.

Reply:
Following our discussions on IRC, i've now opened #373.

Reply:
Thanks! I think you need to link to that and this will be good to go too c:

Reply:
With #373 now merged, i've updated this PR to refer to the subsection added, about how to ignore packages.